apps/web/lib/events/EventManager.ts

```diff
@@ -238,14 +239,20 @@ export default class EventManager {
       },
     });
 
+    const reminderDeletes = prisma.attendeeReminder.deleteMany({
+      where: {
+        bookingUid: booking.uid,
+      },
+    });
+
```

Comment:
To delete a booking we have to delete all other records connected to it, this deletes any reminders that are associated with that booking